taxonomies/ingredients.txt

```diff
@@ -9689,6 +9684,34 @@ es:queso suizo
 it:formaggio svizzero, formaggi svizzeri
 # fr:fromage-suisse has 21 products in 3 languages @2018-10-27
 
+<en:sauce
+en:cheese sauce
+es:salsa de queso
+fi:juustokastike
+fr:sauce au fromage
+nl:kaassaus
+sv:ostsås
+wikidata:en:Q27972032
+# ingredient/en:cheese-sauce has 425 products in 2 languages @2020-06-07
+
```

Comment:
You could delete this. Should be covered by processing?

Reply:
oh, the addition was just moving it out of the named-cheeses section. there's a deletion of the same just below.

Reply:
I was wondering how I could check whether moving ingredients to processing will work other than just pushing to live and hoping, but falling foul of the unit tests just now made me realise I could write ones to check the changes I've made do what I expect, and run them locally on my machine.